### PR TITLE
Feature/atom input 

### PR DESCRIPTION
**Pull request by juanmaguitar**
A.K.A. MoleculeFieldInput

### CODE REVIEW

**Comment by juanmaguitar** [comment]
Online demo can be checked at → https://sui-components-feature-atom-input.now.sh/workbench/atom/input/demo

File: components/atom/input/src/index.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+const AtomInput = ({
+  id,
+  label,
+  successText,
+  errorText,
+  helpText,
+  inline,
+  ...props
+}) => {
+  const newProps = {...props}
```
Comment (jcger) [inline components/atom/input/src/index.js]: this is the same as `const newProps = props`, you could use `props` straight forward

File: components/atom/input/package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "npx rimraf ./lib && npx mkdirp ./lib && npm run build:js && npm run build:styles",
+    "build:js": "../../../node_modules/.bin/babel --presets sui ./src --out-dir ./lib",
+    "build:styles": "../../../node_modules/.bin/cpx './src/**/*.scss' ./lib"
+  },
+  "dependencies": {
+    "@s-ui/component-dependencies": "1",
+    "@s-ui/react-form-input": "1.6.0",
+    "@s-ui/react-molecule-field": "1.1.0"
```
Comment (jcger) [inline components/atom/input/package.json]: Please set only the major version in order to not have to update this package every time

File: components/atom/input/README.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
+# AtomInput
+
+
+`AtomInput` is a component that wraps a composition of Label + Textarea + Validations  Messages. 
+
+
+## Installation
+
+```sh
+$ npm install @s-ui/@s-ui/react-atom-input --save
```
Comment (davidbarna) [inline components/atom/input/README.md]: typo `@s-ui/@s-ui/react-atom-input` => `@s-ui/react-atom-input`

Comment (davidbarna) [reply components/atom/input/README.md]: On line 18 as well

File: components/atom/input/src/index.js (hunk 1 above)
Comment (davidbarna) [reply components/atom/input/src/index.js]: I don't think so because `props` is immutable.

File: components/atom/input/src/index.scss [hunk 4]
```diff
@@ -0,0 +1,7 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+@import "~@s-ui/react-molecule-field/lib/index.scss";
+@import "~@s-ui/react-form-input/lib/index.scss";
+
+.sui-AtomInput {
```
Comment (davidbarna) [inline components/atom/input/src/index.scss]: Remove this unnecessary code. @imports will be executed on build and not used.
(don't remove the file)

**Review by davidbarna — CHANGES_REQUESTED** [review]

**Review by davidbarna — APPROVED** [review]

**Review by jcger — APPROVED** [review]

**Pull request merged**